Spring-Boot/Spring-Boot-Submission/src/main/java/tw/waterball/judgegirl/springboot/submission/controllers/SubmissionRejudgeController.java

```diff
@@ -0,0 +1,37 @@
+package tw.waterball.judgegirl.springboot.submission.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tw.waterball.judgegirl.submission.domain.usecases.RejudgeAllSubmissionsUseCase;
+import tw.waterball.judgegirl.submission.domain.usecases.RejudgeSubmissionUseCase;
+
+/**
+ * @author - c11037at@gmail.com (snowmancc)
+ */
+@CrossOrigin
+@RestController
+@AllArgsConstructor
+@RequestMapping("/api/rejudge")
```

Comment:
this is not RESTFul

the rejudge api should be
`/api/submissions/judges`

its request body consists of `problemId` and the `bag query`